src/EventSubscriber/MailNotificationSubscriber.php

```diff
@@ -79,8 +79,10 @@ public function notifyRegistrationAdded(RegistrationAddedEvent $event): void
 
         $confirmation = $event->getRegistration()->getPerson() === $this->user ? 'bevestiging' : 'bericht';
         $title = "Aanmeld$confirmation ".$activity->getName();
+        $organizer = $event->getRegistration()->getPerson();
```

Comment:
The variable `$organizer` is introduced here. However, the e-mail is directed at the participant of an activity. Please update the name of this variable accordingly.